packages/client-sdk-web/src/cache-client.ts

```diff
@@ -51,6 +51,10 @@ export class CacheClient extends AbstractCacheClient implements ICacheClient {
   public get configuration(): Configuration {
     return this._configuration;
   }
+
+  public static create(cacheClientProps: CacheClientProps): CacheClient {
```

Comment:
the example code that we are creating this for always calls `await` on this, so I think the function will need to be `async`.

I know the linter will complain about this, so you will probably need to add one of those `eslint-disable` pragmas, and a comment explaining why.